t/admin/global-rules2.t

```diff
@@ -61,3 +61,91 @@ __DATA__
     }
 --- response_body
 {"action":"get","count":0,"node":{"dir":true,"key":"/apisix/global_rules","nodes":{}}}
+
+
+
+=== TEST 2: set global rule
+--- config
+    location /t {
+        content_by_lua_block {
+            local json = require("toolkit.json")
+            local t = require("lib.test_admin").test
+            local code, message, res = t('/apisix/admin/global_rules/1',
+                 ngx.HTTP_PUT,
+                [[{
+                    "plugins": {
+                        "proxy-rewrite": {
+                            "uri": "/"
+                        }
+                    }
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+                ngx.say(message)
+                return
+            end
+
+            res = json.decode(res)
+            res.node.value.create_time = nil
+            res.node.value.update_time = nil
+            ngx.say(json.encode(res))
+        }
+    }
+--- response_body
+{"action":"set","node":{"key":"/apisix/global_rules/1","value":{"id":"1","plugins":{"proxy-rewrite":{"uri":"/"}}}}}
+--- request
```

Comment:
There is no need to repeat the two sections below?

Reply:
I want to add a test cases for non-empty resources like `TEST 3`, so put a new global rules.

Reply:
But you already set them (--- request,  --- no_error_log) in https://github.com/apache/apisix/pull/6155/files#diff-150a3eabdca95c99b2f057a9981111fd3dc0f7c830999549b60ce791265319c3R28

Reply:
Got it, I have removed request and error log phrase.